src/Calendar/Calendar.js

```diff
@@ -453,6 +453,7 @@ class Calendar extends Component {
             this.props.localizedText.show12PreviousYears : this.props.localizedText.previousMonth;
         const nextButtonLabel = this.state.showYears ?
             this.props.localizedText.show12NextYears : this.props.localizedText.nextMonth;
+        const compact = this.props?.compact || false;
```

Comment:
`this.props` should never be undefined and is this necessary or can we just use `this.props.compact` below?